pkg/scalers/metrics_api_scaler.go

```diff
@@ -81,12 +82,12 @@ func NewMetricsAPIScaler(config *ScalerConfig) (Scaler, error) {
 
 		// TODO: this will add a transport for all of the HTTP clients.
```

Comment:
We can remove this TODO now, right?

Reply:
yup, done in https://github.com/kedacore/keda/pull/1251/commits/833043a665b27a8909e3667fe016481efd288d78